property.ts

```diff
@@ -40,6 +41,7 @@ export const checkProperties = (
   seed: number | undefined,
   path: string | undefined,
   runs: number | undefined,
+  endOnFailure: boolean,
```

Comment:
Shouldn't this be `bail` to match _our_ DSL? Internally, it can be mapped to whatever fast-check expects. This prevents us from exposing implementation details, avoiding a [leaky abstraction](https://www.joelonsoftware.com/2002/11/11/the-law-of-leaky-abstractions/).

```suggestion
  bail: boolean,
```